packages/react-strict-dom/postcss/bundler.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+const babel = require('@babel/core');
+const reactStrictBabelPreset = require('react-strict-dom/babel-preset');
+
+// Creates a stateful bundler for processing StyleX rules using Babel.
+module.exports = function createBundler() {
+  const styleXRulesMap = new Map();
+
+  // Determines if the source code should be transformed based on the presence of StyleX/RSD imports.
+  function shouldTransform(sourceCode) {
+    return (
+      sourceCode.includes('stylex') || sourceCode.includes('react-strict-dom')
+    );
+  }
+
+  // Transforms the source code using Babel, extracting StyleX rules and storing them.
+  async function transform(id, sourceCode, babelConfig, options) {
+    const { isDev, shouldSkipTransformError } = options;
+    const { code, map, metadata } = await babel
+      .transformAsync(sourceCode, {
+        filename: id,
+        caller: {
+          name: 'react-strict-dom/postcss-plugin',
+          platform: 'web',
```

Comment:
`platform: 'web'` is necessary for RSD Preset to process source code with StyleX